stylegan_code_finder/train_with_lightning.py

```diff
@@ -71,22 +71,27 @@ def main(args: argparse.Namespace):
     else:
         multiprocessing_strategy = 'ddp_find_unused_parameters_false'
 
-    model_checkpoint_callback = ModelCheckpoint(save_top_k=3, monitor='val_loss',
+    model_checkpoint_callback = ModelCheckpoint(save_top_k=3, monitor='val_handwriting_recall', mode='max',
                                                 dirpath=Path(config['log_dir'], 'checkpoints'),
-                                                filename="segmentation-{epoch:02d}-{val_loss:.2f}")
+                                                filename="segmentation-{epoch:02d}-{val_handwriting_recall:.2f}")
     callbacks = [EarlyStopping(monitor='val_loss', patience=config['patience']), model_checkpoint_callback]
 
     if 'max_iter' in config:
         config['epochs'] = 1000  # pytorch lightning default
     else:
         config['max_iter'] = -1
 
+    if config['debug']:
```

Comment:
for this if statement one could use an inline if: `validation_interval = 1.0 if config['debug'] else 0.1`. It's shorter but still readable. (However one might flip the condition so that the "standard" value is at the front)